lib/pingurl.js

```diff
@@ -24,12 +24,20 @@ exports.ping = (pingUrl, inputData, timeout, timeoutWait) => {
     let pinger = new HttpPing(timeout, timeoutWait);
     let opts = {
       url: pingUrl,
-      host: parsed.host,
+      host: parsed.hostname,
```

Comment:
Don't think we've actually hit this, but `parsed.host` includes the port number, where `hostname` does not.